lesson2/homework/helloworld/src/main/java/Account.java

```diff
@@ -36,8 +36,8 @@ public void deposit_money(long deposit){
 
 class CheckingAccount extends Account {
```

Comment:
Generally you should only have one class in each file, so CheckingAccount should be in a new file.